packages/shared/src/libs/enums/exception-message.enum.ts

```diff
@@ -3,6 +3,8 @@ const ExceptionMessage = {
 	CONTRIBUTOR_MERGE_FAILED: "Failed to merge contributor.",
```

Comment:
As I mentioned earlier, we are merging contributors, so let's rename this to 
```suggestion
	CONTRIBUTOR_MERGE_FAILED: "Failed to merge contributors.",
```